src/RecyclableMemoryStreamManager.cs

```diff
@@ -457,7 +457,7 @@ private long RoundToLargeBufferSize(long requiredSize)
         {
             if (this.UseExponentialLargeBuffer)
             {
-                int pow = 1;
+                long pow = 1;
```

Comment:
This caught me up for a bit as I still write a lot of Windows c++. Hopefully I can submit as PR next time, still waiting on legal to clear open source contribs for my team 🙄